contracts/core/interfaces/ICore.sol

```diff
@@ -133,6 +133,18 @@ interface ICore {
     )
         external;
 
+    /**
+     * Converts user's components into Set Token's held directly in Vault
```

Comment:
`Set Tokens`

Reply:
What does held directly in Vault mean? Who does the SetToken show is the owner and what does the Vault balances show?